e_imports.go

```diff
@@ -43,6 +43,7 @@ xargs go list -find -f '{{if (and
 */
 
 import (
+	_ "github.com/alecthomas/kingpin/v2"
```

Comment:
Note: this requires https://github.com/gravitational/teleport.e/pull/1402 to be true.